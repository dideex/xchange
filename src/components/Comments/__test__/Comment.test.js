import React from 'react'
import Component from '../Comment'
import {mount} from 'enzyme'

describe('Comments component', () => {
  it('Clean markup', () => {
    const data = {
      photo: '',
      name: '',
      role: '',
      message: '',
    }
    const wrapper = mount(<Component {...data} />)
    expect(wrapper.html()).toMatchSnapshot()
  })

  it('Markup with data', () => {
    const data = {
      photo: 'Some test photo',
      name: 'Some test name',
      role: 'Some test Role',
      message: 'Some test message',
    }
    const wrapper = mount(<Component {...data} />)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
