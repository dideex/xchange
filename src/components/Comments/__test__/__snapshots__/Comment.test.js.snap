// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments component Clean markup 1`] = `"<article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3></h3><p></p></hgroup></div><div class=\\"css-1lexkey\\"><p><span></span></p></div></article>"`;

exports[`Comments component Markup with data 1`] = `"<article class=\\"css-1104qn0\\"><div class=\\"css-1p5u6wl\\"><img src=\\"Some test photo\\" alt=\\"Avatr\\" class=\\"css-v9h0cw\\"><hgroup><h3>Some test name</h3><p>Some test Role</p></hgroup></div><div class=\\"css-1lexkey\\"><p><span>Some test message</span></p></div></article>"`;
